mmv1/products/managedkafka/Cluster.yaml

```diff
@@ -0,0 +1,151 @@
+--- !ruby/object:Api::Resource
```

Comment:
This file needs a copyright notice like: https://github.com/GoogleCloudPlatform/magic-modules/pull/10928

Reply:
Done.